backend/tasks/training_packs/decrypt.py

```diff
@@ -0,0 +1,203 @@
+            num_addition += 16
+        extension = bytes((num_addition * "\x00").encode())
+        s = s + extension
+        return s
+
+    @staticmethod
+    def _unpad(s):
+        return s[:-ord(s[len(s) - 1:])]
+
+
+def create_shot(ball_x=-182.3192, ball_y=4039.0, ball_z=262.1, ball_vel_p=8100, ball_vel_y=-14287,
+                ball_vel_r=-828, ball_vel_mag=1439.8,
+                car_x=0.0, car_y=0.0, car_z=30.0, car_rot_p=0, car_rot_y=16384, car_rot_r=0):
+    time_limit = Property("FloatProperty", "TimeLimit", 10.0)
+    shot = [{'ObjectArchetype': 'Archetypes.Ball.Ball_GameEditor', 'StartLocationX': ball_x,
+             'StartLocationY': ball_y, 'StartLocationZ': ball_z, 'VelocityStartRotationP': ball_vel_p,
+             'VelocityStartRotationY': ball_vel_y, 'VelocityStartRotationR': ball_vel_r,
+             'VelocityStartSpeed': ball_vel_mag},
+            {'ObjectArchetype': 'Archetypes.GameEditor.DynamicSpawnPointMesh', 'LocationX': car_x,
+             'LocationY': car_y, 'LocationZ': car_z, 'RotationP': car_rot_p, 'RotationY': car_rot_y,
+             'RotationR': car_rot_r},
+            {'IsPC': True, 'LocationX': -599.9999, 'LocationY': -700.0001, 'LocationZ': 529.9955,
+             'RotationP': -1952, 'RotationY': 14037, 'RotationR': 0}, None]
+    return {'TimeLimit': time_limit, 'data': shot}
+
+
+def load_pack(fn):
+    with open(fn, "rb") as f:
+        chunk_size = f.read(4)  # CRC stuff
+        # size = struct.unpack("<I", chunk_size)
+
+        check = f.read(4)  # CRC stuff
+        byte = f.read()
+        # chunk_size_int = struct.unpack("<I", chunk_size)[0]
+        # check_int = struct.unpack("<I", check)[0]
+
+        aes = AESCipher(AES_KEY)
+
+        result = aes.decrypt_bytes(byte)
+
+        parsed_properties = parse(io.BytesIO(result))
+    return parsed_properties
+
+
+def encrypt_test_pack():
+    with open('test.pack', 'rb') as f:
+        recreated = f.read()
+
+    aes = AESCipher(AES_KEY)
+    # # Re-encrypt
+    padded = aes._pad(recreated)
+
+    encrypted = aes.encrypt_bytes(padded)
+    #
+    check_int_generated = struct.pack("<I", create_crc(encrypted))
+    size_int_generated = struct.pack("<I", len(encrypted))
+
+    file = size_int_generated + check_int_generated + encrypted
+    with open("test.Tem", "wb") as output:
+        output.write(file)
+
+
+def reserialize(parsed_properties: List[Property]):
+    recreated = serialize(parsed_properties)
+
+    with open('test.pack', 'wb') as f:
+        f.write(recreated)
+
+    aes = AESCipher(AES_KEY)
+    # # Re-encrypt
+    padded = aes._pad(recreated)
+
+    with open('padded.pack', 'wb') as f:
+        f.write(padded)
+
+    encrypted = aes.encrypt_bytes(padded)
+    #
+    check_int_generated = struct.pack("<I", create_crc(encrypted))
+    size_int_generated = struct.pack("<I", len(encrypted))
+
+    file = size_int_generated + check_int_generated + encrypted
+    return file
+
+
+if __name__ == '__main__':
+    encrypt_test_pack()
```

Comment:
If this is a test can we put it in tests.py?
Or some place else where we can ignore it for coverage purpose